ext/sst/sst_responder_subcomponent.cc

```diff
@@ -99,17 +108,68 @@ SSTResponderSubComponent::handleTimingReq(
 void
 SSTResponderSubComponent::init(unsigned phase)
 {
-    if (phase == 1) {
-        for (auto p: responseReceiver->getInitData()) {
-            gem5::Addr addr = p.first;
-            std::vector<uint8_t> data = p.second;
-            SST::Interfaces::StandardMem::Request* request = \
-                new SST::Interfaces::StandardMem::Write(
-                    addr, data.size(), data);
-            memoryInterface->sendUntimedData(request);
+    if (phase == 0) {
+        // Added support for MPI send and recv. We have to split and send
+        // gem5's data in phases to SST.
+        // get the size of this memory.
+        // We are using a MemBackdoor to get the data to restore from gem5.
+        gem5::MemBackdoorPtr data;
+        responseReceiver->getBackdoor(data);
+        assert(data->readable());
+
+        // this can be a huge number
```

Comment:
What does this comment mean???

Reply:
I'll fix this in the next patch. The comment is not really relevant but I'll try to word this in a better way.